internal/payloads/instance_types_payload.go

```diff
@@ -19,13 +19,10 @@ func (s *InstanceTypeResponse) Render(_ http.ResponseWriter, _ *http.Request) er
 	return nil
 }
 
-func NewListInstanceTypeResponse(sl *[]types.InstanceTypeInfo) []render.Renderer {
-	sList := *sl
-	list := make([]render.Renderer, 0, len(sList))
-	for i := range sList {
-		list = append(list, &InstanceTypeResponse{
-			Identifier: string(sList[i].InstanceType),
-		})
+func NewListInstanceTypeResponse(sl *[]clients.InstanceType) []render.Renderer {
+	list := make([]render.Renderer, 0)
```

Comment:
This will be a tad faster:

```suggestion
	list := make([]render.Renderer, 0, len(sl))
```